app/models/topic.rb

```diff
@@ -0,0 +1,4 @@
+class Topic < ApplicationRecord
+  has_one_attached :image
```

Comment:
Since we are API only, we should consider using `active-storage-base64` gem here